metadata.yaml

```diff
@@ -26,3 +28,12 @@ requires:
 provides:
   kafka-client:
     interface: kafka_client
+
+storage:
+  log-data:
+    type: filesystem
+    description: Directories where the log data is stored
+    minimum-size: 50M
```

Comment:
The specs was kinda asking for 1TB... 

Is there any way we could set this? Should setting be done in bundles maybe?

Reply:
That will be addressed in future PRs. This is for basic functionality at this stage.